core/settings.py

```diff
@@ -201,3 +196,23 @@
     profiles_sample_rate=1.0,
     environment=ENV or "local",
 )
+
+# Enable / Disable Azure Auth
+if os.environ.get("AZURE_AUTH_ENABLED", "true") == "true":
```

Comment:
We want to be careful here because we don't want to accidentally disable auth because of a misconfiguration. I think that checking for an explicit value of "false" is a better approach.

Reply:
Makes sense, ill amend.